saleor/dashboard/templates/dashboard/order/modal_ship_delivery_group.html

```diff
@@ -1,16 +1,21 @@
 {% load i18n %}
+{% load bootstrap %}
+{% load materializecss %}
 
-<div class="modal-content">
-    <h5>{% trans "Ship delivery group " %}#{{ group.pk }}</h5>
+<form id="ship-delivery-group-form" method="post" role="form" action="{% url 'dashboard:ship-delivery-group' order_pk=order.pk group_pk=group.pk %}" class="form-async">
+    {% csrf_token %}
 
-    <p>{% trans "Ship selected delivery group to the following address:" %}</p>
-    {% include 'dashboard/includes/_address.html' with address=group.order.shipping_address only %}
-</div>
+    <div class="modal-content">
+        <h5>{% trans "Ship delivery group " %}#{{ group.pk }}</h5>
```

Comment:
Is showing a DB identifier here improving usability? Did I want to ship group `#1653` or `#1635`?